shared/teams/new-team/wizard/new-team-info.tsx

```diff
@@ -18,22 +31,22 @@ const NewTeamInfo = () => {
   const teamWizardState = Container.useSelector(state => state.teams.newTeamWizard)
 
   const [name, setName] = React.useState(teamWizardState.name)
-  const [teamNameTakenError, setTeamNameTakenError] = React.useState<string | null>(null)
+  const [teamNameTakenStatus, setTeamNameTakenStatus] = React.useState<number>(0)
   const [teamNameTaken, setTeamNameTaken] = React.useState(false)
-  const checkTeamNameTaken = Container.useRPC(RPCTypes.teamsUntrustedTeamExistsRpcPromise)
+  const checkTeamNameTaken = debounce(Container.useRPC(RPCTypes.teamsUntrustedTeamExistsRpcPromise), 100)
```

Comment:
🤔Does this work? I'd assume you'd have to memo it; otherwise it's a new debounced function every time it renders.

Reply:
It does! I tried with a much larger debounce and it was clearly debouncing.